packages/chrome.extensions.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,64 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+
+  $extensions = @(
+    # MetaMask
+    'nkbihfbeogaeaoehlefnkodbefgpgknn'
+    # Phantom
+    'bfnaelmomeimhlpmgjnjophhpkkoljpa'
+    # BNB Chain Wallet
+    'fhbohimaelbohpjbbldcngcnapndodjp'
+    # Avira Password Manager
+    'caljgklbbfbcjjanaijlacgncafpegll'
+    # KeePassXC-Browser
+    'oboonakemofpalcgghocfoadofidjkkk'
+    # Yoroi
+    'ffnbelfdoeiohenkjibnmadjiehjhajb'
+    # XDEFI Wallet
+    'hmeobnfnfcmdkdcmlblgagmfpfboieaf'
+    # ...
+  )
+
+  $regKeyPath = "HKLM:\SOFTWARE\WOW6432Node\Policies\Google\Chrome\ExtensionInstallForcelist"
+  $updateUrl = "https://clients2.google.com/service/update2/crx"
+
+  New-Item -Path $regKeyPath -Force -ea 0 | Out-Null
+  $valueName = 1
+  Foreach ($extensionId in $extensions)
+  {
+    New-ItemProperty -Path "$regKeyPath" -Name "$valueName" -Type String -Value "$extensionId;$updateUrl" -Force -ea 0 | Out-Null
+    $valueName += 1
+  }
+
+  $chromePath = "${Env:ProgramFiles}\Google\Chrome\Application\chrome.exe"
+  $extensionsDir = "${Env:LocalAppData}\Google\Chrome\User Data\Default\Extensions"
+
+  $loaded = $false
+  while (-not $loaded)
+  {
+    # Restart Chrome.
+    Stop-Process -Force -Name Chrome -ea 0
+    $chromeId = (Start-Process -FilePath $chromePath -passthru).id
+
+    # Wait for the extensions to be loaded.
+    Start-Sleep -Seconds 120
+    Stop-Process -Force -Id $chromeId | Out-Null
```

Comment:
I have tested this and when reopening Chrome some times I get the following error:
![image](https://github.com/user-attachments/assets/faee6ffd-b84d-4ec6-bdd8-455cc1d99956)

Is there something we can do to mitigate this issue? :thinking: @mandiant/vms ideas?

Reply:
Done.